api/v1alpha3/virtualmachinereplicaset_types.go

```diff
@@ -96,14 +130,22 @@ type VirtualMachineReplicaSetStatus struct {
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 }
 
+func (rs *VirtualMachineReplicaSet) GetConditions() []metav1.Condition {
+	return rs.Status.Conditions
+}
+
+func (rs *VirtualMachineReplicaSet) SetConditions(conditions []metav1.Condition) {
+	rs.Status.Conditions = conditions
+}
+
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:scope=Namespaced,shortName=vmreplicaset
+// +kubebuilder:resource:scope=Namespaced,shortName=vmrs
```

Comment:
Multiple short name aliases are allowed if you want to retain the original as well.